app/controllers/users.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+const bcrypt = require('bcryptjs');
+const logger = require('../logger');
+const User = require('../models').User;
+
+exports.create = (req, res, next) => {
+  const userObj = req.body || {};
+  User.isPwdLengthGreater(userObj, 8)
+    .then(() => {
+      bcrypt.hash(userObj.password, 10).then(hashedPwd => {
```

Comment:
you dont need to wait for the user.ispwdfLengthGreater for calling this one, you can do a promise.all with both